parametric/test_headers_tracestate_dd.py

```diff
@@ -131,6 +131,8 @@ def test_headers_tracestate_dd_propagate_samplingpriority(test_agent, test_libra
 
     # 4) Sampled = 1, tracestate[dd][s] <= 0
     # Result: SamplingPriority = 1
+    # TODO(stephen.belanger): Is this correct? -1 input through dd headers
+    # results in -1 out the other end but here expects -1 to become 1?
```

Comment:
Yes, we're prioritize the tracestate's sampled flag. So as long as it is 1, we will propagate our sampling priority with a value >= 1. If the incoming DD sampling priority was >=1, use the value. Otherwise, set to 1